project.owasp.yaml

```
@@ -0,0 +1,27 @@
+audience:
+  - builder
+leaders:
+  - name: Jim Manico
+    email: jim@owasp.org
+    github: jmanico
+  - name: Jeff Ichnowski
+    email: jeff.ichnowski@owasp.org
```

While this hasn't officially been updated - I think this should be:
```suggestion
  - name: Jim Manico
    email: jim@owasp.org
    github: jmanico
  - name: Jeremy Long
    email: jeremy.long@owasp.org
```

I'll need to check with Jim and Starr regarding this.

---

@jeremylong @jmanico @mamicidal please adjust it as needed.